src/Features/CSharpTest/ConvertNamespace/ConvertNamespaceRefactoringTests.cs

```diff
@@ -608,7 +608,8 @@ public async Task TestConvertToBlockScopedInCSharp9()
                 FixedCode = @"
 namespace $$N
 {
-}",
+}
+",
```

Comment:
can you add a test that validates when tehre isn't a trailing end of line?

Reply:
➡️ Updated